pci/src/vfio.rs

```diff
@@ -1717,7 +1719,7 @@ impl VfioPciDevice {
         if !self.iommu_attached {
             self.container
                 .vfio_dma_map(iova, size, user_addr)
-                .map_err(VfioPciError::DmaMap)?;
+                .map_err(|e| VfioPciError::DmaUnmap(e, self.bdf.to_string()))?;
```

Comment:
I think this variant should be of `DmaMap` 🤔, no?

Reply:
yep.. good catch.

Reply:
I will help you trigger the workflows after you have this fixed 😉

Reply:
pushed the fix.

Reply:
Noted, and let's wait for more input on this PR, shall we 😉